release-notes/swan-lake-beta2-release-note.md

```diff
@@ -36,6 +36,27 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Improvements
 
+The `Configurable` feature is improved to support variables with the `union` types through the TOML syntax. 
```

Comment:
This needs to be a bullet point, right?

Reply:
This is same as beta 1 release note. Do We need to change?

Reply:
This breaks the flow with the rest of the points, please check https://github.com/ballerina-platform/ballerina-release/blob/master/release-notes/swan-lake-beta2-release-note.md#improvements. Currently looks like it belongs to "Support for recursive tuple types has been introduced with this release.".

We either use bullet points or subheadings. Here, I guess we can go with bullet points.

Reply:
will send a fix in a separate PR